.github/workflows/basic.yml

```diff
@@ -181,7 +181,7 @@ jobs:
         build_type: [Debug, Release]
         compiler: [{c: cl, cxx: cl}]
         pool_tracking: ['ON', 'OFF']
-        shared_library: ['OFF']
+        shared_library: ['ON', 'OFF']
```

Comment:
perhaps you should also enable windows builds in `proxy_lib.yml`...?

Reply:
It is impossible - there is no `LD_PRELOAD` mechanism on Windows ...